src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -198,7 +198,10 @@ private static String findAndroidStudio(@Nullable Project project) {
 
     final FlutterSdk flutterSdk = FlutterSdk.getFlutterSdk(project);
     if (flutterSdk != null) {
-      String androidSdkLocation = flutterSdk.queryFlutterConfig("android-studio-dir", false);
+      String androidSdkLocation = flutterSdk.queryFlutterConfig("android-studio-dir", true);
```

Comment:
The `useCachedValue` flag will return a value if one is cached; if there's no cached value, it'll call through to querying `flutter config`; I don't think we need the 2nd call to `queryFlutterConfig()` here.

Reply:
The File.exists() is key here. The internet is full of advice about using `flutter config --android-studio-dir` which will invalidate the cache if done while the IDE is active.

Reply:
ack